source/languages/en/riak/theory/concepts/Replication.md

```diff
@@ -76,6 +76,20 @@ increase it, you might need to force read repair (more on that below).
 Overwritten objects and newly stored objects will automatically be
 replicated to the correct number of nodes.  </div>
 
+## Changing the N value (`n_val`)
+
+While raising the value of N for a bucket or object shouldn't cause
+problems, it's important that you never lower N. If you do so, you can
+wind up with dead, i.e. unreachable data. This can happen because
+objects' preflists, i.e. lists of [[vnodes]] responsible for the object,
+can end up 
```

Comment:
This unfinished sentence is still present in the master branch, as well on http://docs.basho.com/riak/2.1.1/theory/concepts/Replication/

Shall I create an issue?